lib/rubocop/minitest/support.rb

```diff
@@ -1,10 +1,259 @@
 # frozen_string_literal: true
 
 # Require this file to load code that supports testing using Minitest.
+# rubocop:disable Metrics/ClassLength
 
 require 'rubocop'
 require 'minitest/autorun'
 require 'minitest/pride'
```

Comment:
This should probably go when others are using this as well

Reply:
I don't follow, do you want me to remove this line entirely?

Reply:
If I decide to use this all my logs are suddenly absolutely fabulous since it includes the minitest pride plugin

Reply:
Ok, I think I understand a bit better, I've moved it to the `test_helper`